layers/meta-resin-qemu/recipes-core/images/resin-image.bbappend

```diff
@@ -0,0 +1,5 @@
+IMAGE_FSTYPES_append_qemuall = " resin-sdcard"
+
+# Customize resin-sdcard
+RESIN_IMAGE_BOOTLOADER_qemuall = ""
+RESIN_BOOT_PARTITION_FILES_qemuall = ""
```

Comment:
Indeed. After that I do add support for another qemu target. This is just a template commit.